WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -514,40 +513,107 @@ - (void)refreshPullToRefresh
     }
 }
 
+#pragma mark - No Results Views
+
+- (void)initNoResultsViews
```

Comment:
> could it be mistaken for an initializer?

Sorry, I'm not seeing what could be confusing. I don't get what one would think this was an initializer of. I can rename it to whatever though. I changed it from `configureNoResultsView` because it's not actually configuring, and _that_ confused _me_.  😄  Would `initializeNoResultsViews` be better? Or is `initialize` the part you take issue with?

Reply:
I was just thinking about ObjC's pattern of `initWithFoo:`, `initWithBar:` for initializers and wondered if the similarity would give someone pause at a glance.  I might just be overthinking it but wanted to ask :)